File: Computabilidade-e-Complexidade-de-Algoritmos-main/java/6.java
public class AFNPeloMenosUmZero {
    public static boolean peloMenosUmZero(String s) {
        return s.contains("0");
    }

    public static void main(String[] args) {
        System.out.println(peloMenosUmZero("101"));  // True
        System.out.println(peloMenosUmZero("111"));  // False
    }
}

File: Computabilidade-e-Complexidade-de-Algoritmos-main/golang/6.go
package main
import "fmt"

func peloMenosUmZero(s string) bool {
    for _, char := range s {
        if char == '0' {
            return true
        }
    }
    return false
}

func main() {
    fmt.Println(peloMenosUmZero("101"))  // True
    fmt.Println(peloMenosUmZero("111"))  // False
}
